packages/electron-scripts/src/commands/build.ts

```diff
@@ -118,8 +118,9 @@ export async function build(options: IBuildCommandOptions): Promise<void> {
         outDir,
     });
 
-    const builderConfig = fs.readJsonFileSync(fs.join(basePath, electronBuilderConfigFileName)) as Configuration;
-
+    const configFullPath = fs.resolve(fs.join(basePath, electronBuilderConfigFileName));
+    const builderConfig: Configuration = await import(configFullPath) as Configuration;
```

Comment:
when we dynamically import a commonjs file, we get it as a default export.
so `const {default: builderConfig} = (await import(configFilePath)) as {default: Configuration}`